docs/guide/05_plans/charges/overview.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+In addition to the plan price, you have the flexibility to charge your customers based on their usage. This allows you to create charges that align with specific metrics, such as the number of *API calls*, *active users*, *transactions*, *compute time*, and more. These additional charges are directly tied to the billable metrics defined earlier.
+
+## Adding Usage-Based Charges to a Plan
+To incorporate usage-based charges into a plan, you can utilize existing billable metrics. This enables you to offer "pay-as-you-go" features.
+
+## Key Information about Usage-Based Charges
+Here are some important details to consider regarding usage-based charges:
+
+### Invoice Timing
```

Comment:
You mention `cadence` above and here `timing`, are we talking of the same concept here?